### PR TITLE
Set qx.ui.table.model.Filtered class as deprecated

### PR DESCRIPTION
**Pull request by woprandi**
For https://github.com/qooxdoo/qooxdoo/issues/9204

### CODE REVIEW

**Review by patrick-fischer-hirschstein — APPROVED** [review]

**Review by johnspackman — CHANGES_REQUESTED** [review]
please can you restore the documentation in the comments?  Although this class is being deprecated, there's no need to loose the documentation or otherwise modify it

**Comment by woprandi** [comment]
@johnspackman you're right

**Comment by woprandi** [comment]
@johnspackman updated

**Review by johnspackman — APPROVED** [review]
thank you :)

**Review by cajus — APPROVED** [review]

**Review by level420 — APPROVED** [review]

File: framework/source/class/qx/ui/table/model/Filtered.js [hunk 1]
```diff
@@ -68,6 +68,10 @@ search.addListener("changeValue", function(e) {
 
 this.getRoot().add(search, {top : 500, left : 10});
 </pre>
+ *
+ * @deprecated {6.0} You should use
+ * <a href='http://developer.mozilla.org/docs/Web/JavaScript/Reference/Objets_globaux/Array/filter' target='_blank'>Array.filter</a>
```
Comment (level420) [inline framework/source/class/qx/ui/table/model/Filtered.js]: The Link to the mozilla developper network does not seem to work

**Review by level420 — CHANGES_REQUESTED** [review]

File: framework/source/class/qx/ui/table/model/Filtered.js (hunk 1 above)
Comment (woprandi) [inline framework/source/class/qx/ui/table/model/Filtered.js]: @level420 it works for me

Comment (johnspackman) [inline framework/source/class/qx/ui/table/model/Filtered.js]: doesnt work for me either

Comment (level420) [inline framework/source/class/qx/ui/table/model/Filtered.js]: @woprandi the "Objet_globaux" part int he URL seems to be for the french link. My link gets rewritten to start with https://developer.mozilla.org/de/... (german) and is suspect @johnspackman s link gets rewritten to https://developer.mozilla.org/en/ . So wie need the full englisch path in the URL

Comment (johnspackman) [inline framework/source/class/qx/ui/table/model/Filtered.js]: I think you meant https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter - it looks like MDN localises the URLs and unless the locale is specified explicitly it's not reliable

Comment (woprandi) [inline framework/source/class/qx/ui/table/model/Filtered.js]: @level420 Oh sorry, I'll fix that

**Review by level420 — APPROVED** [review]

**Comment by level420** [comment]
Documentation only fix. Merging.
Thank you @woprandi

**Pull request merged**